azure-pipelines-integration.yml

```diff
@@ -3,9 +3,7 @@ trigger:
 - master
 - master-vs-deps
 - dev16.0-preview2
-- dev16.0-preview2-vs-deps
 - dev16.0-preview3
-- dev16.0-preview3-vs-deps
```

Comment:
Why isn't this deleting the other -vs-deps, because it'll break those too, right?

Reply:
master-vs-deps and newer are expected to work with integration tests running against 2019